### DISCUSSION TITLE
break two line in save

### DISCUSSION THREAD

**Discussion by WreewanMorhee** [prettier/prettier#15894] [Q&A] [answered]
How should I break two line ?

this is default (one line break): 
<img width="228" alt="Screenshot 2024-01-08 at 5 32 35 PM" src="https://github.com/prettier/prettier/assets/33398071/30098c7a-febc-40e4-abbe-664a79e01947">

and I want this (two line break):
<img width="223" alt="Screenshot 2024-01-08 at 5 33 04 PM" src="https://github.com/prettier/prettier/assets/33398071/78f2719f-6db5-48f9-a97b-0dc1c91735f3">

**Comment by lulunac27a** [comment, marked as answer]
There is no way to break two empty lines in Prettier. But you can use comments as sections to break two empty lines.
